src/libraries/Microsoft.Extensions.Caching.Abstractions/src/PACKAGE.md

```diff
@@ -2,43 +2,51 @@
 
 <!-- A description of the package and where one can find more documentation -->
 
+Microsoft.Extensions.Caching.Abstractions offers the basics to create and use caching in your applications.
 
+This library doesn’t hold the actual caches, but it defines how caches should behave.
+With this, you or other libraries can build various types of caches that all apps can use in the same way, whether the data is kept in memory, in files, or even across a network.
 
 ## Key Features
 
 <!-- The key features of this package -->
 
-*
-*
-*
+* Basic rules and tools for building different kinds of caches.
+* Makes it easier to create or use caches in your apps, whether they're simple or complex.
```

Comment:
This sounds vague to me. How does it make it easier? What's simple and what's complex? Consider removing it.
```suggestion
```